geopandas/geoseries.py

```diff
@@ -751,11 +751,27 @@ def notnull(self):
         return self.notna()
 
     def fillna(self, value=None, method=None, inplace=False, **kwargs):
-        """Fill NA values with a geometry (empty polygon by default).
+        """
+        Fill NA values with geometry (or geometries).
+
+        ``method`` is currently not implemented.
+
+        Parameters
+        ----------
+        value : Geometry or GeoSeries, default None
+            If None is passed, NA values will be filled with GEOMETRYCOLLECTION EMPTY.
+            If ``pygeos.Geometry`` or a shapely geometry object is passed, it will be
```

Comment:
```suggestion
            If scalar geometry object is passed, it will be
```
This will make our life easier when deprecating pygeos. And avoids the confusion as you cannot pass pygeos.Geometry to a shapely-backed GeoSeries.